packages/docusaurus-theme-classic/src/theme/DocSidebarItem/Category/index.tsx

```diff
@@ -72,20 +72,35 @@ function useCategoryHrefWithSSRFallback(
 }
 
 function CollapseButton({
+  isActive,
   categoryLabel,
   onClick,
 }: {
+  isActive: boolean;
   categoryLabel: string;
   onClick: ComponentProps<'button'>['onClick'];
 }) {
-  return (
+  return isActive ? (
+    <button
+      aria-label={translate(
+        {
+          id: 'theme.DocSidebarItem.collapseCategoryAriaLabel',
+          message: "Collapse sidebar category '{label}'",
+          description: 'The ARIA label to collapse the sidebar category',
+        },
+        {label: categoryLabel},
+      )}
+      type="button"
+      className="clean-btn menu__caret"
+      onClick={onClick}
+    />
+  ) : (
     <button
       aria-label={translate(
         {
-          id: 'theme.DocSidebarItem.toggleCollapsedCategoryAriaLabel',
-          message: "Toggle the collapsible sidebar category '{label}'",
-          description:
-            'The ARIA label to toggle the collapsible sidebar category',
+          id: 'theme.DocSidebarItem.expandCategoryAriaLabel',
+          message: "Expand sidebar category '{label}'",
+          description: 'The ARIA label to expand the sidebar category',
```

Comment:
we could have a ternary only on the aria-label prop instead of having 2 different button JSX here